src/app/services/utility.service.ts

```diff
@@ -0,0 +1,9 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+
+export class UtilityService {
+  show = false;
```

Comment:
Why are we using this service for storing a value only?

Reply:
in next steps I would write other functions in utility service which I want to use in multiple modules